recipe/script.js

```diff
@@ -0,0 +1,42 @@
+function validTextColor(strColor) {
+    const instanceStyle = new Option().style;
+    instanceStyle.color = strColor; 
+    return instanceStyle === strColor;
+}
+
+let backgroundColor = function() {
+    let userColor = prompt("What background color do you want?", 'rgb(164, 247, 222)');
+    if(!validTextColor(userColor)){
+      console.log("This is not a valid input");
```

Comment:
you'll want to return after this `console.log` so that the`document.body.style.backgroundColor` won't get set to an invalid color